pkg/jobs/jobs.go

```diff
@@ -329,7 +329,7 @@ func (j *Job) FractionProgressed(ctx context.Context, progressedFn FractionProgr
 		}
 		if fractionCompleted < 0.0 || fractionCompleted > 1.0 {
 			return errors.Errorf(
-				"Job: fractionCompleted %f is outside allowable range [0.0, 1.0] (job %d)",
+				"job: fractionCompleted %f is outside allowable range [0.0, 1.0] (job %d)",
```

Comment:
please change to
`job %d: fractionCompleted %f is outside allowable range [0.0, 1.0]`